### PR TITLE
✨ Ads: 1WorldOnline ad integration

### PR DESCRIPTION
**Pull request by AlexanderKadygrob**
Implemented 1WorldOnline ad integration

### CODE REVIEW

**Comment by AlexanderKadygrob** [comment]
Can't add reviewers here. Please check, @lannka, @zhouyx

File: ads/1wo.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
+/**
+ * Copyright 2017 The AMP HTML Authors. All Rights Reserved.
```
Comment (zhouyx) [inline ads/1wo.js]: nit: 2019

File: ads/1wo.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
+<!---
+Copyright 2017 The AMP HTML Authors. All Rights Reserved.
```
Comment (zhouyx) [inline ads/1wo.md]: nit: 2019

**Comment by AlexanderKadygrob** [comment]
> Looks great. few nits. Please also link to your 1wo.md from amp-ad.md. Thank you

Done

**Comment by zhouyx** [comment]
Thank you @AlexanderKadygrob for adding the support. PR merged.

**Pull request merged**